client/src/elements/atoms/icon-button.js

```diff
@@ -1,10 +1,16 @@
 import { bindable } from "aurelia-framework";
 
 export class IconButton {
+  @bindable click;
   @bindable icon;
   @bindable code;
   @bindable size;
   @bindable iconSize;
   @bindable tabindex;
+  @bindable disabled;
   @bindable type = "button";
+
+  handleClick(event) {
+    debugger;
```

Comment:
The debugger should probably be removed?

Reply:
damn, sure.